### PR TITLE
Fix tailwind and use yarn

### PR DESCRIPTION
**Pull request by CotyJ**
Fixes errors in setup with Tailwind using Yarn

```yarn run v1.22.22
$ tailwindcss -i ./app/assets/stylesheets/application.tailwind.css -o ./app/assets/builds/application.css
/bin/sh: 1: tailwindcss: Permission denied
error Command failed with exit code 126.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <link rel="apple-touch-icon" href="/icon.png">
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    <%= stylesheet_link_tag "application" %>
```
Comment (malakada) [inline app/views/layouts/application.html.erb]: We actually already have this just above on line 16. :) If you can remove please?

**Review by malakada — CHANGES_REQUESTED** [review]
Too many application.css

**Review by atvo912 — APPROVED** [review]

**Review by malakada — APPROVED** [review]
tyty! 🏆

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (CotyJ) [reply app/views/layouts/application.html.erb]: Done, why did you add this line on my commit? :O

**Pull request merged**